docs/reference.rst

```diff
@@ -240,6 +240,11 @@ hierarchical separator.
   active slots, so that the inactive one can be selected as the update target.
   The parent slot is referenced using the form ``<slot-class>.<idx>``.
 
+``allow-mounted=<true/false>``
+  Setting this entry ``true`` tells RAUC to that the slot may be
+  updated even if it is already mounted. Such a slot can only be
+  updated by a custom install hook.
```

Comment:
nit: to my understanding this should be:

```suggestion
  updated even if it is already mounted. Such a slot can be
  updated only by a custom install hook.
```

We might also do other things with the slot than updating, but this is not the point here ;)